src/main/java/me/kodysimpson/simpapi/utility/builders/ItemBuilder.java

```diff
@@ -0,0 +1,192 @@
+package me.kodysimpson.simpapi.utility.builders;
+
+import org.bukkit.ChatColor;
+import org.bukkit.Color;
+import org.bukkit.DyeColor;
+import org.bukkit.Material;
+import org.bukkit.enchantments.Enchantment;
+import org.bukkit.inventory.ItemFlag;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.ItemMeta;
+import org.bukkit.inventory.meta.LeatherArmorMeta;
+import org.bukkit.inventory.meta.SkullMeta;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Easily create itemstacks, without messing your hands.
+ * <i>Note that if you do use this in one of your projects, leave this notice.</i>
+ * <i>Please do credit me if you do use this in one of your projects.</i>
+ *
+ * @author NonameSL
+ */
+// TODO: TEST
+public class ItemBuilder {
+    private final ItemStack is;
+
+    /**
+     * Create a new ItemBuilder from scratch.
+     *
+     * @param m The material to create the ItemBuilder with.
+     */
+    public ItemBuilder(Material m) {
+        this(m, 1);
```

Comment:
This does no checks for `null` or `Material.AIR`. Both will result in many methods in this class failing. As the intended audience of this library contains newcomers to the Spigot programming world, these are non-obvious footguns. Consider adding checks to protect consumers.